resources/liquibase/001-initial-schema.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" logicalFilePath="001-initial-schema.xml">
+  <changeSet id="001" author="guseyn">
+    <sql>
+      CREATE TABLE "user" (
+        id SERIAL PRIMARY KEY,
+        login VARCHAR(64) NOT NULL,
+        avatar_url VARCHAR(100) NOT NULL,
+        created TIMESTAMPTZ DEFAULT now() NOT NULL
+      );
+      CREATE INDEX idx_user1 ON "user" (login);
+    </sql>
+    <sql>
+      CREATE TABLE project (
+        id SERIAL PRIMARY KEY,
+        submitter INT NOT NULL REFERENCES "user"(id),
+        title VARCHAR(64) NOT NULL,
```

Comment:
@Guseyn I don't think it's correct. Let's have `coordinates` (e.g. "yegor256/takes") and `platform` ("github", "bitbucket", etc.)

Reply:
@yegor256 okay